src/modules/claims/claims.service.ts

```diff
@@ -472,30 +473,30 @@ export class ClaimsService {
 
     if (registrationTypes.includes(RegistrationTypes.OffChain)) {
       await this.verifyIssuer(claimData.claimType);
+      const vp = await this.issueVerifiablePresentation({
+        subject: sub,
+        namespace: role,
+        version: version.toString(),
+        issuerFields,
+        credentialStatus: credentialStatusOverride,
+        expirationTimestamp: claimExpirationTimestamp,
+      });
+      const credStatus = vp?.verifiableCredential[0]?.credentialStatus;
       const publicClaim: IPublicClaim = {
         did: sub,
         signer: this._signerService.did,
+        credentialStatus: credentialStatusOverride || credStatus,
```

Comment:
I'm not sure we want to add the `credentialStatus` property if both `credentialStatusOverride || credStatus` are undefined/null. I think it would be better in this case if `credentialStatus` wasn't specified? 🤔 

I used this script to test that JS still adds props to objects if the value is undefined. https://playcode.io/new/
```
let i = undefined;
const message = {
  prop: i
}

// Log to console
console.log(message)
console.log('prop' in message)
console.log(message.hasOwnProperty('prop'))
```